components/feed/FeedItem.tsx

```diff
@@ -85,6 +82,14 @@ const FeedItem = ({
               />
             </svg>
           </div>
+          <div>
+            <LikeButton
+              userId={currentUser.id}
+              postId={data.id}
+              likes={data.likes}
```

Comment:
```suggestion
              likes={data.likedIds.length}
```

And then delete the entire `likes` field (less chance that something breaks and it doesn't stay in sync).